### DISCUSSION TITLE
[request] pp

### DISCUSSION THREAD

**Discussion by WindowsMeosu** [gitMiloTilo/peppypedia#39] [General]
**What type of Post Request?**
Map, Users, Clans, etc.
* Feature
* Ranking System

**Why do you think this would be worth adding as a wiki page?**
Is this user known? Is this map overrated or underrated? how big is this clan?
* osu! ranking system 
* Any posts that mention them or are related?**
If so, mention them here:

**Additional context**
Add any other context or screenshots about the post request here.

NOTICE: will be resolved after https://github.com/WindowsMeosu/peppypedia/issues/70 closes

<p>Bypassing full sentences obviously because we all know what pp is (don't put that out of context)</p>

**Comment by WindowsMeosu** [comment]
alright I think I finished gathering enough info

**Comment by qwrmolly** [comment]
branch made

**Reply by WindowsMeosu** [reply]
not the best naming but works

**Comment by WindowsMeosu** [comment]
this issue here: https://github.com/WindowsMeosu/peppypedia/issues/70
